tests/unit/test.utils.js

```diff
@@ -606,4 +606,105 @@ describe('isCompatible', () => {
       )
     ).toBe(true);
   });
+
+  it('should be compatible if version is behind flag and no other version is available', () => {
+    expect(
+      isCompatible(
+        getBCD({
+          foo: {
+            __compat: {
+              support: { firefox: { flags: [], version_added: true } },
+            },
+          },
+        }),
```

Comment:
Nit, I'm wondering if a small test helper could be useful here to make these new test cases a bit shorter (without having to engage a fight with "prettier" ;-))

Reply:
I'm not quite sure what part could be done with a helper, so please elaborate.

Reply:
> I'm not quite sure what part could be done with a helper, so please elaborate.

@freaktechnik Sure thing!

I was thinking to just a simple test helper to make the single test case a bit less verbose (while keeping "prettier" still happy ;-)), nothing too "fancy".

The helper I was thinking of looks more or less like the following:

```js
  const getBCDFirefoxCompat = (
    firefoxSupportData,
    { apiName = 'foo', appName = 'firefox' } = {}
  ) =>
    getBCD({
      [apiName]: { __compat: { support: { [appName]: firefoxSupportData } } },
    });
```

Then it should be possible to reduce most of the test case to something like:

```js
    expect(
      isCompatible(
        getBCDFirefoxCompat({ version_added: '60' }),
        'foo',
        60,
        'firefox'
      )
    ).toBe(true);
```
or
```js
    expect(
      isCompatible(
        getBCDFirefoxCompat(
          { version_added: '61' },
          { apiName: 'foo', appName: 'firefox_android' }
        ),
        'foo',
        60,
        'firefox_android'
      )
    ).toBe(false);
```
or
```js
    expect(
      isCompatible(
        getBCDFirefoxCompat([
          { version_added: '60' },
          { flags: [], version_added: '59' },
        ]),
        'foo',
        60,
        'firefox'
      )
    ).toBe(true);
```